### PR TITLE
(MODULES-6468) - PDK Convert

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
```
Comment (pmcmaw) [inline .gitlab-ci.yml]: This file is not needed.

File: .pdkignore [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.*.sw[op]
```
Comment (pmcmaw) [inline .pdkignore]: This file is not needed.

**Comment by pmcmaw** [comment]
There are 2 files that are not needed.
When removed I will get this PR merged.

**Comment by david22swan** [comment]
Done

File: update_report.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* Report generated by PDK at 2018-03-06 15:18:11 +0000 */
```
Comment (pmcmaw) [inline update_report.txt]: This whole file can be removed

**Comment by david22swan** [comment]
Done, and Squished

**Pull request merged**